.circleci/config.yml

```diff
@@ -74,6 +74,7 @@ jobs:
           command: dockerize -wait tcp://localhost:5432 -timeout 1m
 
       - run: mix test
+      - run: cd assets/ && npm run lint
```

Comment:
Will this fail the build?

Reply:
Yes! Eslint returns a non-zero exit code when there are linting errors.